src/copyPath.js

```diff
@@ -0,0 +1,9 @@
+const assocPath = require('ramda/src/assocPath')
+const curry = require('ramda/src/curry')
+const path = require('ramda/src/path')
+
+// copyPath :: [String] -> [String] -> { k: v } -> { k: v }
+const copyPath = (fromPath, toPath, obj) =>
+  assocPath(toPath, path(fromPath, obj), obj)
```

Comment:
I like how this is written.  The style has parity with that of the [`copyProp`](https://github.com/articulate/funky/blob/master/src/copyProp.js) implementation.  But I also wanted to note an alternative using `funky.combineWith`:
```js
const copyPath = (fromPath, toPath) =>
  combineWith(assocPath(toPath), path(fromPath))

module.exports = uncurryN(3, copyPath)
```
But that said, I like yours better.  It's easier to understand the intent at a glance.